extensions/parquet/table/src/main/java/io/deephaven/parquet/table/location/ParquetPushdownUtils.java

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright (c) 2016-2025 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.parquet.table.location;
+
+import io.deephaven.time.DateTimeUtils;
+import org.apache.parquet.column.statistics.Statistics;
+import org.apache.parquet.schema.ColumnOrder;
+import org.apache.parquet.schema.PrimitiveType;
+
+import java.time.Instant;
+
+/**
+ * Utility methods for Parquet pushdown operations.
+ */
+final class ParquetPushdownUtils {
+    private static final long NANOS_PER_SECOND = DateTimeUtils.SECOND;
+
+    /**
+     * Converts nanoseconds from the Epoch to an {@link Instant}.
+     */
+    static Instant epochNanosToInstant(final long nanos) {
+        return Instant.ofEpochSecond(nanos / NANOS_PER_SECOND, nanos % NANOS_PER_SECOND);
+    }
+
+
+    static boolean areStatisticsUsable(final Statistics<?> statistics) {
+        if (statistics == null || !statistics.hasNonNullValue()) {
```

Comment:
Would it be usable for certain filters (I'm thinking match) if we had only null values?